dapps/marketplace/src/components/growth/Action.js

```diff
@@ -104,13 +104,28 @@ function Action(props) {
     externalLink = `https://twitter.com/intent/tweet?text=${encodeURIComponent(
       getContentToShare(props.action, props.locale)
     )}`
+  } else if (type === 'FacebookShare') {
+    buttonLink = undefined
+    foregroundImgSrc = 'images/growth/facebook-icon.svg'
+    title = fbt('Share this on Facebook', 'RewardActions.postThisOnFacebook')
+    externalLink = [
```

Comment:
@sparrowDom Just a thought, What do you think about moving these external links stuff to growth rules?

Reply:
interesting... thinking about it I would rather keep it here since I think this is implementation detail rather than configuration one. For example Origin Wallet depends on Twitter share dialog to match `twitter.com/intent/tweet` when it is intercepting the dialog on Android and triggering native share. 

I would rather see that changing that would require code changes on both sides instead of configuration on dapp/growth part and code in Origin Wallet